build.rs

```diff
@@ -44,6 +44,11 @@ fn main() -> anyhow::Result<()> {
                     "tests/wast/spec/proposals/multi-value",
                     wast_processor,
                 )?;
+                test_directory_module(
+                    spectests,
+                    "tests/wast/spec/proposals/reference-types",
+                    wast_processor,
+                )?;
```

Comment:
Because reference types are fully stabilized, we don't need this and it's likely that the tests in there are out of date and failing in wasmparser or the semantics are out of date. #2268 is fully testing reference types